cmd/admin-handlers-site-replication.go

```diff
@@ -334,6 +343,7 @@ func (a adminAPIHandlers) SiteReplicationStatus(w http.ResponseWriter, r *http.R
 		opts.Users = true
 		opts.Policies = true
 		opts.Groups = true
+		opts.ILMExpiryRules = true
```

Comment:
you mean, if ILM expiry replication not enable, don't report their replication status in `mc admin replicate status ALIAS`?  Currently it would show status as below if ILM expiry replication is disabled
```
ILM Expiry Rules replication status:
●  0/1 ILM Expiry Rules in sync
```